docs/Onboarding Docs/01_Razor_Basics.md

```diff
@@ -0,0 +1,52 @@
+﻿# Razor Basics
+
+Razor is a templating language used in ASP.NET for creating dynamic web pages. It's not a programming language itself, but a server-side markup syntax for embedding server-side code (C# or VB.NET) into HTML.
```

Comment:
```suggestion
Razor is a templating language used in ASP.NET for creating dynamic web pages. It's not a programming language itself, but a markup syntax for embedding code (C# or VB.NET) into HTML.
```

I would remove the words "server-side" from this, as Blazor can be purely client side.